PWGEM/Dilepton/TableProducer/skimmerOTS.cxx

```diff
@@ -85,14 +100,14 @@ struct skimmerOTS {
     if (mRunNumber == bc.runNumber()) {
       return;
     }
-
+    zorro.setBaseCCDBPath(cfgCcdbPathZorro.value);
```

Comment:
This is not needed anymore, because zorro objects have been moved to common directory.